src/api/tests/fixtures/collect_records.py

```diff
@@ -134,32 +134,32 @@ def valid_collect_record(
 ):
     observations = [
         dict(
-            count=10,
+            count="10",
```

Comment:
By making these strings, we're testing both the casting and the resulting values, is that it?